pkg/iac-providers/kustomize/v3/types.go

```diff
@@ -0,0 +1,29 @@
+package kustomizev3
+
+import (
+	"fmt"
+)
+
+// KustomizeV3 struct
+type KustomizeV3 struct{}
+
+const (
+	// YAMLExtension yaml
+	YAMLExtension = "yaml"
+	// YAMLExtension2 yml
+	YAMLExtension2 = "yml"
+	// KustomizeFileName kustomization
+	KustomizeFileName = "kustomization"
+)
+
+// KustomizeFileNames returns the valid extensions for k8s (yaml, yml, json)
+func KustomizeFileNames() []string {
+	return []string{
+		getFullFileName(KustomizeFileName, YAMLExtension),
+		getFullFileName(KustomizeFileName, YAMLExtension2),
+	}
+}
+
+func getFullFileName(file, ext string) string {
```

Comment:
Would `AddSuffix` be a better name for this functon? And would it make sense to add it to `utils` package?

Reply:
yes, makes sense.